charts/incubator/nvidia-gpu-exporter/values.yaml

```diff
@@ -0,0 +1,80 @@
+image:
+  repository: utkuozdemir/nvidia_gpu_exporter
+  pullPolicy: IfNotPresent
+  tag: 1.2.0@sha256:cc407f77ab017101ce233a0185875ebc75d2a0911381741b20ad91f695e488c7
+securityContext:
+  container:
+    privileged: true
+    readOnlyRootFilesystem: false
+    runAsUser: 0
+    runAsGroup: 0
+service:
+  main:
+    ports:
+      main:
+        protocol: http
+        port: 9835
+workload:
+  main:
+    type: DaemonSet
+    podSpec:
+      containers:
+        main:
+          args:
+            - --web.listen-address
+            - :{{ .Values.service.main.ports.main.port }}
+            - --web.telemetry-path
+            - /metrics
+            - --nvidia-smi-command
+            - nvidia-smi
+          probes:
+            liveness:
+              path: /
+              port: main
+            readiness:
+              path: /
```

Comment:
Maybe it has a health endpoint? or `/metrics` ?

Reply:
/metrics endpoint added.